SKILL.tsv

```diff
@@ -5706,9 +5706,9 @@ SKILL_20151102_005705	$Increases the damage dealt on an enemy with [Incineration
 SKILL_20151102_005706	$Bloodletting: Hemostasis
 SKILL_20151102_005707	$Increases the cycle time of the HP cost by 0.2 seconds per attribute level while [Bloodletting] is active.
 SKILL_20151102_005708	$Bloodletting: Replace SP
-SKILL_20151102_005709	$Cuts SP instead of HP at a 10% chance per attribute level while the effect per tick of [Bloodletting] is active.
-SKILL_20151102_005710	$Increases the chance by 5% per attribute level to block Lv3 or less abnormal statuses for 10 seconds after receiving the effect of [Fumigate].
-SKILL_20151102_005711	$Purifies the area for 10 seconds within the range of [Fumigate]. Increases the Poison property resistance of allies by 10%.
+SKILL_20151102_005709	$While [Bloodletting] effect is active, each tick has a 10% chance per every attribute level to reduce SP instead of HP.
```

Comment:
While the [Bloodletting] effect is active, each tick has a 10% chance per attribute level to reduce SP instead of HP.

per / every - Both mean the same thing and so one of them is redundant. Use 'per' since that term is used elsewhere in SKILL.